src/device-registry/config/constants.js

```diff
@@ -7,6 +7,10 @@ const devConfig = {
   REDIS_PORT: process.env.REDIS_PORT,
   KAFKA_BOOTSTRAP_SERVERS: process.env.KAFKA_BOOTSTRAP_SERVERS_DEV,
   KAFKA_TOPICS: process.env.KAFKA_TOPICS_DEV,
+  SCHEMA_REGISTRY: process.env.SCHEMA_REGISTRY_DEV,
```

Comment:
Awesome stuff! Love that we are following the convention of uppercase while separating the ENVs accordingly....
For items which have the same values in all ENVs, please consider using the defaultConfig (ALONE) below...thanks

Reply:
@Baalmart, all necessary changes have been made